requirements.txt

```diff
@@ -1,32 +1,45 @@
 alembic==0.9.6
+astroid==1.5.3
```

Comment:
pleace clean requirements.txt of pylint packages

Reply:
done